### ISSUE TITLE
Cannot change Bundle Identifier

### ISSUE THREAD

**Issue by dangthaison91** [e-sites/Natrium#13]
Everything is changed for `Info Plist` and `Natrium.swift`, but `PRODUCT_BUNDLE_IDENTIFIER` is not changed in build settings. I will try with version 6.1.0.

**- Version: 6.1.1**

**.natrium.yml**:
```yaml
settings:
  class_name: BuildConfiguration

environments:
  - Dev
  - Staging
  - Production

natrium_variables:
  BundleID:
    Dev: com.domain.XXX.dev
    Staging: com.domain.XXX.staging
    Production: com.domain.XXX

target_specific:
  Target_XXX:
    xcconfig:
      PRODUCT_BUNDLE_IDENTIFIER:
        Dev: "#{BundleID}"
        Staging: "#{BundleID}"
        Production: "#{BundleID}"

plists:
  "AppXXX/Configuration/AppXXX-Info.plist":
    CFBundleDisplayName:
      Dev: Dev AppXXX
      Staging: Staging AppXXX
      Production: AppXXX
```

**- Natrium log:** 

```ruby
[environments]
[5:38:46 PM]: ▸      - Dev
[5:38:46 PM]: ▸      - Staging
[5:38:46 PM]: ▸      - Production
[5:38:46 PM]: ▸   [settings]
[5:38:46 PM]: ▸     class_name = BuildConfiguration
[5:38:46 PM]: ▸   [target_specific:AppXXX]
[5:38:46 PM]: ▸     xcconfig
[5:38:46 PM]: ▸       PRODUCT_BUNDLE_IDENTIFIER = "com.domain.XXX"
[5:38:46 PM]: ▸   [natrium_variables]
[5:38:46 PM]: ▸     AppName = "AppName 1"
[5:38:46 PM]: ▸   [plists]
[5:38:46 PM]: ▸     AppXXX/Configuration/AppXXX-Info.plist
[5:38:46 PM]: ▸       CFBundleDisplayName = Staging AppXXX
[5:38:46 PM]: ▸     AppXXX/Configuration/AppXXX-Debug-Info.plist
[5:38:46 PM]: ▸       CFBundleDisplayName = Staging XXX Debug
```

**Comment by dangthaison91** [comment]
Seem that Natrium xcconfig variable was not overridden ?

**Comment by basvankuijck** [comment]
Could you try:
```
target_specific:
  VinID:
    xcconfig:
      PRODUCT_BUNDLE_IDENTIFIER: "#{BundleID}"
```
?

**Comment by dangthaison91** [comment]
@basvankuijck  It didn't work. I found that if my configuration not include **xcconfig**, natrium will not create **xcconfig file** for each build configuration.

`target_specific` will override xcconfig only if file is exited.

**Comment by basvankuijck** [comment]
Resolved in `v6.1.2`
